src/intervals/trigonometric_functions.jl

```diff
@@ -4,9 +4,13 @@ half_pi(::Type{Float64})  = @floatinterval(pi/2)
 two_pi(::Type{Float64})   = @floatinterval(2pi)
 half_pi(::Type{BigFloat}) = @biginterval(pi/2)
 two_pi(::Type{BigFloat})  = @biginterval(2pi)
-
 half_pi{T<:AbstractFloat}(x::T) = half_pi(T)
 
+pi_interval{T<:Real}(::Type{T}) = get_pi(Float64)
+range_atan2{T<:Real}(::Type{T}) = Interval(-pi_interval(T).hi, pi_interval(T).hi)
+half_range_atan2(::Type{Float64}) = @floatinterval(-pi/2, pi/2)
+half_range_atan2(::Type{BigFloat}) = @biginterval(-pi/2, pi/2)
```

Comment:
This seems to be the correct way of defining the range, in the sense that it correctly changes if the interval precision changes.